unittests/v1.2.0/test_report_generation.py

```diff
@@ -0,0 +1,87 @@
+from compliance_suite.report_runner import *
+
+auth_types = ["none", "basic", "bearer", "passport"]
+
+def test_constructor():
```

Comment:
I suggest removing this test since it only verifies the creation of the default empty constructor, which seems unrelated to the current context of this test file. This test appears to be arbitrary and does not contribute to the purpose of the file.